### DISCUSSION TITLE
Hiding inactive questionnaires

### DISCUSSION THREAD

**Discussion by jtk2ho** [KL-Psychological-Methodology/ESMira#17] [Questions] [answered]
Hello everyone,

I am conducting an EMA study where participants complete a repeated questionnaire multiple times per day when notified. I have set the "filter" for a repeated questionnaire so that the questionnaire can only be filled out once per notification. I would like participants to only be able to complete the questionnaire when notified, but I noticed that in the app the questionnaire can still be accessed and submitted when viewing "inactive" questionnaires. Is there a way to restrict the questionnaire so that participants cannot access it at all unless they are completing it through a study notification?

Thanks in advance!

**Comment by JodliDev** [comment]
Hello,

Unfortunately that is not possible yet. We almost implemented it a while ago but did not need it in the end and also wanted to wait in case questionnaires are hidden from the dashboard by mistake.

But it is quite easy to add. I will implement it and let @SelinaDev know when its done, so she can schedule it for the next release.

**Reply by jtk2ho** [reply]
That would be very helpful, thank you!

**Comment by JodliDev** [comment, marked as answer]
I have added the feature. We need a bit more testing to make sure everything works and then it will be rolled out with the next release.

**Reply by jtk2ho** [reply]
Thank you!

**Reply by SelinaDev** [reply]
This feature is now available, both in the clients in the stores as well as in the new version of the server.

**Reply by jtk2ho** [reply]
Amazing, appreciate it!
